### PR TITLE
ut for toasts and transition

### PR DESCRIPTION
**Pull request by jimmymadrigal**
Transition was not possible to test, toasts testing done right

### CODE REVIEW

File: packages/toast/tests/unit/ui-toast.spec.ts [hunk 1]
```diff
@@ -1,12 +1,194 @@
...
+
+  beforeEach(async () => {
+    wrapper = mount(UiToastContainer)
+
+    service = ToastMessageQueueService.singletonInstance
+    service.clear()
+  })
+
+  it('should render when props are set', () => {
+    const wrapper = mount(UiToastContainer)
```
Comment (alejandronavarroGL) [inline packages/toast/tests/unit/ui-toast.spec.ts:20]: I wonder if this wrapper could be the same declared in line 13

Comment (jimmymadrigal) [reply packages/toast/tests/unit/ui-toast.spec.ts]: solved, thanks

File: packages/toast/tests/unit/ui-toast.spec.ts [hunk 2]
```diff
@@ -1,12 +1,194 @@
...
+      },
+    })
+    service?.add({
+      headerText: 'some header',
+      message: 'some message',
+    })
+    await wrapper2.vm.$nextTick()
+
+    const toastBody = wrapper2.find(`div.${theme.cssClass.body} a`)
+    console.log()
```
Comment (alejandronavarroGL) [inline packages/toast/tests/unit/ui-toast.spec.ts:97]: I think console.log is not needed

Comment (jimmymadrigal) [reply packages/toast/tests/unit/ui-toast.spec.ts]: solved, thanks

**Pull request merged**